modules/ROOT/nav.adoc

```diff
@@ -63,18 +63,18 @@ include::partial$nav-migration.adoc[]
  ** xref:on-error-scope-concept.adoc[On-Error Error Handlers]
  ** xref:until-successful-scope.adoc[Until Successful Scope]
 * xref:mule-app-strategies.adoc[General Configurations]
- ** xref:reconnection-strategy-about.adoc[Reconnection Strategies]
-  *** xref:reconnection-strategy-reference.adoc[Reconnection Strategy Reference]
- ** xref:shared-resources.adoc[Shared Resources]
- ** xref:sources-reference.adoc[Sources Reference]
- ** xref:streaming-about.adoc[Streaming in Mule Apps]
-  *** xref:streaming-strategies-reference.adoc[Streaming Strategies Reference]
- ** xref:error-handling.adoc[Error Handling]
-  *** xref:mule-error-concept.adoc[Mule Errors]
- ** xref:configuring-properties.adoc[Configuring Properties]
+ ** xref:configuring-properties.adoc[Configure Properties]
   *** xref:mule-app-properties-to-configure.adoc[Configure Property Placeholders for Mule Apps]
   *** xref:mule-app-properties-system.adoc[System Properties]
   *** xref:custom-configuration-properties-provider.adoc[Custom Configuration Properties Provider]
+ ** xref:error-handling.adoc[Error Handling]
+  *** xref:mule-error-concept.adoc[Mule Errors]
+ ** xref:streaming-about.adoc[Streaming in Mule Apps]
+  *** xref:streaming-strategies-reference.adoc[Streaming Strategies Reference]
+ ** xref:shared-resources.adoc[Shared Resources]
+ ** xref:reconnection-strategy-about.adoc[Reconnection Strategies]
+  *** xref:reconnection-strategy-reference.adoc[Reconnection Strategy Reference]
+ ** xref:sources-reference.adoc[Sources Reference]
  ** xref:target-variables.adoc[Enrich Data with Target Parameters]
```

Comment:
Shoot, I left out target-variables from the rankings: It should go right above reconnection-strategy links.